.github/workflows/updates-testing-nightly.yml

```diff
@@ -0,0 +1,22 @@
+name: fedora-updates-testing
+on:
+  schedule:
+    - cron: '0 1 * * *'
+  # can be run manually on https://github.com/cockpit-project/cockpit-podman/actions
+  workflow_dispatch:
+    permissions:
+      statuses: write
```

Comment:
This is part of `workflow_dispatch:` now, which won't work. It must be at the top level.